OctaneVSPlugin/ViewModel/DetailedItemViewModel.cs

```diff
@@ -286,7 +286,7 @@ public bool OnlyDefaultFieldsAreShown
                     viewModels.Add(new CommentViewModel(comment));
                 }
 
-                _commentViewModels = new ObservableCollection<CommentViewModel>(viewModels.OrderByDescending(c => c.CreationTime));
+                _commentViewModels = new ObservableCollection<CommentViewModel>(viewModels.OrderByDescending(c => DateTime.Parse(c.CreationTime)));
```

Comment:
just out of curiosity, what changed here?

Reply:
@atothhpe Before the change we would do a string ordering. That meant if we had 2 elements: 08/10/2017 and 05/03/2018, then 08/10/2017 would be the first element. My change is to do a Date comparison which is what we needed in the first place